.travis.yml

```diff
@@ -86,17 +86,18 @@ jobs:
         - scripts/skip-doc-change.sh || travis_terminate 0;
         - make image-cephcsi || travis_terminate 1;
         - scripts/travis-functest.sh v1.17.0 || travis_terminate 1;
-
-    - name: cephcsi on Arm64
-      arch: arm64
-      script:
-        - scripts/skip-doc-change.sh || travis_terminate 0;
-        - make image-cephcsi || travis_terminate 1;
-        # No CI test job is availabe for Arm64 now due to below issues
-        # - k8s-csi sidecar images for Arm64 are not available
-        # - Travis Arm64 CI job runs inside unprivileged LXD which blocks
-        #   launching minikube test environment
-        - travis_terminate 0    # deploy only on x86
+     # yamllint disable rule:comments-indentation
+     # Disabled build on arm64 as we are facing issue in travis CI
+     # - name: cephcsi on Arm64
+     #   arch: arm64
+     #   script:
+     #     - scripts/skip-doc-change.sh || travis_terminate 0;
```

Comment:
Actually, does this mean `scripts/skip-doc-change.sh` does not return non-zero, so that further checks are still executed?
Fixing this script (or the way its return value is handled) would be better than just disabling all checks.

Reply:
the issue is not with our scripts, this seems Travis CI issue when doing update there is an issue 
https://travis-ci.org/github/ceph/ceph-csi/jobs/664231300#L97-L100

Reply:
ok, needs to rebase on #873 once that has been merged